lib/Web/API.hs

```diff
@@ -8,12 +8,38 @@ import Backend.Vault.Kv
 import CLI.Types
 import Coffer.Directory (Directory)
 import Coffer.Path (EntryPath, Path, QualifiedPath)
-import Data.HashMap.Strict
+import Data.Aeson.Casing (aesonPrefix, camelCase)
+import Data.Aeson.TH (deriveJSON)
+import Data.HashMap.Internal.Strict (HashMap)
 import Data.Text (Text)
 import Entry
 import GHC.Generics (Generic)
 import Servant.API
 
+----------------------------------------------------------------------------
+-- Additional types for WebAPI
+----------------------------------------------------------------------------
+
+data FieldWODate = FieldWODate
```

Comment:
I think `NewField` would be a better name, similar to how you named `NewEntry`.

Reply:
Agree, changed